src/jsonapi-datastore.js

```diff
@@ -11,6 +11,7 @@
     this.id = id;
     this._type = type;
     this._attributes = [];
+    this._associations = [];
```

Comment:
If I understand correctly, `_associations` here refers to the *inverse* associations, right?